### PR TITLE
feat: verification and decoding for simple hash receipts

### PR DESCRIPTION
**Pull request by robinbryce**
Output looks like this:

  {
    "anchor": "f29b54ec5f183ba17a434809bef8a06e457aa63cd99deab3bcb54715901f0910",
    "api_query": "https://app.dev-robin-0.dev.jitsuin.io/archivist/v2/assets/-/events?order_by=SIMPLEHASHV2&proof_mechanism=SIMPLE_HASH&timestamp_accepted_since=2023-06-01T13:33:00Z&timestamp_accepted_before=2023-06-08T11:18:02Z",
    "endOperator": "lt",
    "endTimeRFC3339": "2023-06-08T11:18:02Z",
    "endTimeUnix": 1686223082,
    "eventCount": 1,
    "hashSchemaVersion": 2,
    "proofMechanism": 2,
    "startOperator": "gte",
    "startTimeRFC3339": "2023-06-01T13:33:00Z",
    "startTimeUnix": 1685626380,
    "tenant": "tenant/c8c51acf-5084-48ba-9e76-2c3cd9439818"
  }

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: rkvst_receipt_scitt/v1.py [hunk 1]
```diff
@@ -43,6 +69,10 @@ def main(args=None):  # pragma: no cover
 not supplied the account existence is not verified.
 """,
     )
+    s.add_argument(
+        "--fqdn", help="The fully qualified domanin name of the rkvst deployment",
```
Comment (honourfish) [inline rkvst_receipt_scitt/v1.py:73]: minor: domanin

File: rkvst_receipt_scitt/receipt.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+    b = b[:16]  # the low bytes are zero
+
+    return uuid.UUID(int=int.from_bytes(b, "big"))
+
+
+
+class Receipt:
+    """
+    This class uses the EIP1186 *neutral* receipt format to encode a receipt for an RKVST events.
```
Comment (honourfish) [inline rkvst_receipt_scitt/receipt.py:64]: minor: for an RKVST events.

File: rkvst_receipt_scitt/receipt.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            f"expected 32 bytes for a uuid storage value not {len(b)}"
+        )
+
+    b = b[:16]  # the low bytes are zero
+
+    return uuid.UUID(int=int.from_bytes(b, "big"))
+
+
+
+class Receipt:
```
Comment (honourfish) [inline rkvst_receipt_scitt/receipt.py:62]: Fantastic use of common code :)

**Review by honourfish — APPROVED** [review]

File: rkvst_receipt_scitt/v1.py (hunk 1 above)
Comment (robinbryce) [reply rkvst_receipt_scitt/v1.py]: ta, done

**Review by JAG-UK — APPROVED** [review]
Taking Joe's word for it :-)

**Pull request merged**